ODIN_II/SRC/ast_elaborate.cpp

```diff
@@ -1815,6 +1815,7 @@ ast_node_t* reduce_expressions(ast_node_t* node, sc_hierarchy* local_ref, long*
                     max_size = (long*)calloc(1, sizeof(long));
 
                     if (node->children[1]->type != NUMBERS) {
+                        node->children[1]->types.operation.output_port_width = assignment_size;
```

Comment:
just do at line 1782
```
if(assignment_size) {
    node->types.operation.assignment_size = assignment_size
```

and this should cover it

Reply:
I'll move it inside the IF expression checking the ``num_children > 0``, otherwise it results in segfault since sum nodes don't have any children.